src/line-graph.js

```diff
@@ -57,8 +59,23 @@ export class LineGraph extends HTMLElement {
         })
       }
     });
+
+   
   }
 
+  /**
+   * @name getDataLineColor
+   * @description // create function to return random color array for each line.
+   */
+
+  getDataLineColor(colors) {
+    var arr = [];
+    var arrColor;    
+    colors.split(',').map(value => arr.push(value));
+    arrColor = [...arr, ...d3.schemeCategory10];
```

Comment:
Ahh you gotta love a good use of spread operator lol

Reply:
yeah !!!